app/views/new-overview.html

```diff
@@ -354,8 +354,11 @@ <h2 ng-if="!app">
                 </div>
               </div>
 
-              <!-- service instances appear the same way no matter the filter listBy option -->
-              <div ng-if="overview.filteredServiceInstances.length">
+              <!--
```

Comment:
Curious how you would feel about collapsing some of this down/pushing it up into the controller (might eliminate the need for the comment)?  Otherwise looks good to me.